ui/components/MesherySettings.js

```diff
@@ -326,7 +325,7 @@ class MesherySettings extends React.Component {
                 tab="tabMetrics"
               />
             </Tooltip>
-            <Tooltip title="Choose Performance Test Defaults" placement="top">
+            {/* <Tooltip title="Choose Performance Test Defaults" placement="top">
```

Comment:
Also, add a comment here that says why you have commented this , that is, _functionality of this tab will take on a different shape, until that, keeping it commented_